### ISSUE TITLE
Include the title of the document at the top of the page

### ISSUE THREAD

**Issue by christianp** [chirun-ncl/chirun#113]
A page for a single chapter doesn't have a level 1 header with the page's title, I think because it's visible in the breadcrumbs.
There should be a `<header>` tag containing an `<h1>` with the page's title, so it's read out by a screen reader.
